akka/src/test/scala/anorm/AkkaStreamSpec.scala

```diff
@@ -26,8 +21,8 @@ class AkkaStreamSpec extends org.specs2.mutable.Specification {
     "expose the query result as source" in { implicit ee: EE =>
       withQueryResult(stringList :+ "A" :+ "B" :+ "C") { implicit con =>
         AkkaStream.source(SQL"SELECT * FROM Test", SqlParser.scalar[String]).
-          runWith(Sink.seq[String]) must beEqualTo(Seq("A", "B", "C")).
-          await(0, 5.second)
+            runWith(Sink.seq[String]) must beEqualTo(Seq("A", "B", "C")).
```

Comment:
weird formatting difference, could you check `sbt scalariformFormat test:scalariformFormat`